code/src/components/thought/Likes.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import HeartIcon from '../../assets/heart-icon.png';
+
+const Likes = ({ id, likes, onLiked }) => {
+  const LIKE_URL = `https://happy-thoughts-technigo.herokuapp.com/thougths/${id}/like`;
+
+  const handleClick = () => {
+    fetch(LIKE_URL, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: ''
+    }).then(() => onLiked(id))
```

Comment:
Your onLiked-function updates the like locally. Instead, you could have done a new GET-request in the .then to get the messages from the API directly. That would follow the ide of having just "one source of truth", i.e. the API. But of course your version works as well :)

Reply:
I got inspired by Jennie's lecture. The like is posted so the API gets updated as our unique source of truth. I think that the advantage of this solution is that it limits the number of fetches, which can be a good thing when the number of requests is limited or when the API is "heavy". Not sure that it was necessary in this case though 😊